channels/sa.m3u

```diff
@@ -17,10 +17,12 @@ http://m.live.net.sa:1935/m/tanasuhtv/playlist.m3u8
 http://ayat.live.net.sa:1935/live/ayat/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7aD1yan.jpg" group-title="",Beity 
 http://82.212.74.99:8000/live/8164.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/KRuV5yv.png" group-title="",CITRUSS 
-https://citrusstv.akamaized.net/hls/live/687285/CTV/index.m3u8
```

Comment:
This channel works for me. (Tested in VLC)
<img width="738" alt="Captura de pantalla 2020-02-07 a las 8 27 20" src="https://user-images.githubusercontent.com/7253922/74003443-ad952e80-4983-11ea-9845-62779d24c823.png">

Reply:
It's a duplicate of https://github.com/iptv-org/iptv/blob/master/channels/ae.m3u#L48

Reply:
No, it's not. In fact, it leads to HLS playlist that contains several broadcast options in different quality, including your option.

<img width="766" alt="Captura de pantalla 2020-02-08 a las 10 05 26" src="https://user-images.githubusercontent.com/7253922/74080980-a9881000-4a5a-11ea-8ab4-d94fd3b2ecb2.png">

Reply:
added back